src/text-field/text-field.svelte

```diff
@@ -44,7 +48,7 @@
   {#if multiline}
     <textarea
       {id}
-      bind:value={value}
+      bind:value={internalValue}
       class={inputClass}
       bind:this={inputElement}
       on:input={(e) => dispatch('input', { value: e.target.value, nativeEvent: e })}
```

Comment:
Shouldn't this take the `value` instead of `e.target.value` for the casting to number to be effective?

Reply:
Yep, that's right